pkg/ddevapp/testdata/TestApptypeDetection/drupal6/misc/ahah.js

```diff
@@ -246,7 +246,7 @@ Drupal.ahah.prototype.error = function (response, uri) {
   // Undo hide.
   $(this.wrapper).show();
   // Re-enable the element.
-  $(this.element).removeClass('progess-disabled').attr('disabled', false);
+  $(this.element).removeClass('progress-disabled').attr('disabled', false);
```

Comment:
Again, this is actual Drupal 6 code, no need to correct. I wonder if this bug was ever fixed.